app4java/src/main/java/com/nyxei/pollingus/service/QuestionYesNo.java

```diff
@@ -0,0 +1,27 @@
+package com.nyxei.pollingus.service;
+
+import java.util.ArrayList;
+
+public class QuestionYesNo extends IQuestion {
```

Comment:
Best way for `YesNo question` is to make it implements `IQuestion` and extends `MultipleChoiceQuestion` (which is an abstract class)